src/defaults/json.pipe.js

```diff
@@ -29,6 +28,10 @@ const timing = require('../utils/timing');
 
 /* eslint newline-per-chained-call: off */
 
+function hasNoContent({ content }) {
+  return !(content !== undefined && content.body !== undefined);
```

Comment:
`content.body` could also be null, no ?

why not simply:

```suggestion
  return content && content.body;
```

Reply:
so that you can specify a `''` content.

Reply:
> `content.body` could also be null, no ?

I guess not.